### PR TITLE
drop TypeORM from SQL-only modules

### PR DESCRIPTION
**Pull request by depombo**

### CODE REVIEW

**Review by dfellis — APPROVED** [review]

File: src/modules/interfaces.ts [hunk 1]
```diff
@@ -464,9 +464,14 @@ export class ModuleBase {
...
   init() {
+    this.loadBasics();
+    this.loadTypeORM();
+  }
+
+  loadBasics() {
     if (!this.dirname) {
       this.dirname =
-        path.dirname(callsite()?.[1]?.getFileName?.()) ??
+        path.dirname(callsite()?.[2]?.getFileName?.()) ??
```
Comment (aguillenv) [inline src/modules/interfaces.ts:474]: why this change?

**Review by aguillenv — APPROVED** [review]

File: src/modules/interfaces.ts (hunk 1 above)
Comment (depombo) [reply src/modules/interfaces.ts:474]: this looks at the stack trace and since I added another function call it changed
